docs/quickstart/install-and-use-a-package-using-the-dotnet-cli.md

```diff
@@ -1,40 +1,40 @@
 author: JonDouglas
 ms.author: jodou
-ms.date: 01/23/2018
+ms.date: 08/17/2022
 ms.topic: quickstart
 ---
 
-# Quickstart: Install and use a package using the dotnet CLI
+# Quickstart: Install and use a package with the dotnet CLI
 
-NuGet packages contain reusable code that other developers make available to you for use in your projects. See [What is NuGet?](../What-is-NuGet.md) for background. Packages are installed into a .NET Core project using the `dotnet add package` command as described in this article for the popular [Newtonsoft.Json](https://www.nuget.org/packages/Newtonsoft.Json/) package.
+NuGet packages contain compiled binary code that developers make available for other developers to use in their projects. For more information, see [What is NuGet](../What-is-NuGet.md). This quickstart describes how to install the popular [Newtonsoft.Json](https://www.nuget.org/packages/Newtonsoft.Json) NuGet package into a .NET project by using the [dotnet add package](/dotnet/core/tools/dotnet-add-package) command.
 
-Once installed, refer to the package in code with `using <namespace>` where \<namespace\> is specific to the package you're using. You can then use the package's API.
+You refer to installed packages in code with a `using <namespace>` directive, where `<namespace>` is the package name. You can then use the package's API in your project.
 
 > [!Tip]
-> **Start with nuget.org**: Browsing nuget.org is how .NET developers typically find components they can reuse in their own applications. You can search nuget.org directly or find and install packages within Visual Studio as shown in this article.
+> Browse [nuget.org/packages](https://nuget.org/packages) to find packages you can reuse in your own applications. You can search directly at [https://nuget.org](https://nuget.org/packages), or find and install packages from within Visual Studio. For more information, see [Find and evaluate NuGet packages for your project](../consume-packages/finding-and-choosing-packages.md).
 
 ## Prerequisites
 
-- The [.NET Core SDK](https://www.microsoft.com/net/download/), which provides the `dotnet` command-line tool. Starting in Visual Studio 2017, the dotnet CLI is automatically installed with any .NET Core related workloads.
+- The [.NET SDK](https://www.microsoft.com/net/download), which provides the `dotnet` command-line tool. Starting in Visual Studio 2017, the dotnet CLI automatically installs with any .NET or .NET Core related workloads.
 
 ## Create a project
 
-NuGet packages can be installed into a .NET project of some kind. For this walkthrough, create a simple .NET Core console project as follows:
+You can install NuGet packages into a .NET project. For this walkthrough, create a simple .NET console project by using the dotnet CLI, as follows:
 
-1. Create a folder for the project.
+1. Create a folder named *nuget1* for the project.
```

Comment:
Can we remove the exact name for a folder and tell users to create a folder of their choosing?

Reply:
@JonDouglas - Due to the namespace declaration being necessary, and because we want to present quickstart code for the user, the only easy way to do that is to have them create a named folder. I could use placeholders, but then the code would not run without modification.

Reply:
I'm just saying the previous version was fine. If we keep the folder name, let's please use a more descriptive name than nuget1.

Reply:
@JonDouglas - Changed namespace and project name to Nuget.Quickstart.